placementApp/urls.py

```diff
@@ -5,7 +5,8 @@
 router = routers.DefaultRouter()
 router.register("students", views.StudentViewSet)
 router.register("positions", views.PositionViewSet)
-
+router.register("company", views.CompanyViewSet)
+router.register("company/<int:pk>", views.CompanyViewSet)
```

Comment:
No need to add another url company/<int:pk>  router.register takes care of it by default

Reply:
I know fixed it

Reply:
Rayyan directly use ModelViewSet